backend/x86_binary_emitter.ml

```diff
@@ -73,10 +73,18 @@ type symbol = {
   mutable sy_num : int option; (* position in .symtab *)
 }
 
+module StringTbl = Hashtbl.Make (struct
```

Comment:
As elsewhere, replace this by `module String = Misc.Stdlib.String` and use `String.Tbl`.